src/java.base/share/classes/java/security/Security.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1996, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1996, 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Seems incorrect to have 3 copyright years.

Reply:
Correct, only the beginning and current year.  `1996, 2023, Oracle...`